Src/BootCamp.Chapter/Demo1.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    static class Demo1
+    {
+        public static void Demo()
+        {
+            var Manager = new CredentialsManager(""); 
+            
+            Console.WriteLine("Would you like to register or login");
+
```

Comment:
You could split the whole intro for user in its' own method. Also you don't need a blank line in between every WriteLine- it's all related.

Reply:
changed